custom/icds_reports/sqldata/exports/children.py

```diff
@@ -211,71 +212,86 @@ def columns(self):
                 'Percent of newborns with low birth weight',
                 percent,
                 [
-                    SumColumn('low_birth_weight_in_month'),
-                    SumColumn('weighed_and_born_in_month')
+                    SumColumn('low_birth_weight_in_month',
+                              alias='low_birth_weight_in_month'),
```

Comment:
what does this do?

Reply:
this aliasing allows to reuse this column in the same query and not calculate again.